eol-checker/tests/test_jira.py

```
@@ -33,9 +33,32 @@ def test_init_uses_env_overrides(monkeypatch):
+    flexmock(jira_module).should_receive("Jira").and_return(mock_client)
+
+    assert fetcher.jira is None
+
+
+def test_jira_property_creates_client_once(monkeypatch, fetcher):
+    monkeypatch.setenv("JIRA_USERNAME", "user@redhat.com")
+    monkeypatch.setenv("JIRA_PASSWORD", "secret")
+    mock_client = flexmock(http_status_code=200)
+    mock_client.should_receive("http_status_code_handler").with_args(200).once()
+    flexmock(jira_module).should_receive("Jira").with_args(
+        url=fetcher.jira_url, username="user@redhat.com", password="secret"
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)